baseline.gradle

```diff
@@ -82,6 +82,7 @@ subprojects {
           '-Xep:StrictUnusedVariable:OFF',
           '-Xep:TypeParameterShadowing:OFF',
           '-Xep:TypeParameterUnusedInFormals:OFF',
+          '-Xep:DangerousThreadPoolExecutorUsage:OFF',
```

Comment:
This was needed to add `ThreadPools.newScheduledPool`. For some reason, errorprone doesn't like it when you create a `ScheduledExecutorService`.